components/collapsible-panel/collapsible-panel.js

```diff
@@ -26,7 +27,7 @@ const defaultHeading = 3;
  * @fires d2l-collapsible-panel-expand - Dispatched when the panel is expanded
  * @fires d2l-collapsible-panel-collapse - Dispatched when the panel is collapsed
  */
-class CollapsiblePanel extends FocusMixin(RtlMixin(LitElement)) {
+class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
```

Comment:
We could update the `README.md` to add the new `skeleton` attribute which is inherited from the `SkeletonMixin`

Reply:
I was planning to make a single PR for the documentation changes of this PR and [the new slot](https://github.com/BrightspaceUI/core/pull/3599).